client.go

```diff
@@ -106,6 +106,16 @@ type Volume interface {
 	// Destroy removes the volume and its contents. Note that it does not
 	// safeguard against child volumes being present.
 	Destroy() error
+
+	// GetStreamP2pUrl should be called against volume streaming dest worker. It
+	// returns P2P streaming URL where the other worker can access. This URL will
+	// be sent to source worker by ATC, then source worker can streaming volume
+	// directly to dest worker at the URL.
```

Comment:
Thanks for the re-wording. Took it.

Reply:
> I was confused about why this function was on the `Volume` interface instead of the `Client` interface. I understood why once I read the implementation and saw that a `StreamIn` URL is returned. I think modifying the comment would help clear this up.

I thought over, and moved `GetP2pUrl` out of volume server. Stream-in url is merged into p2p url in client code.